### PR TITLE
Adds color-hex-length rule

### PR DESCRIPTION
**Pull request by BeccaAnderton**
## Suggested rule/changes(s):
* `color-hex-length` : `severity: ERROR`

@netsells/frontend - Please review

### CODE REVIEW

**Comment by rowbi-wan** [comment]
can someone convince me why `#ffffff` is better than `#fff`?

**Comment by rowbi-wan** [comment]
can we at least reduce it to a warning rather than a scary ERROR?

**Comment by BeccaAnderton** [comment]
No, because you'll just ignore the warning 😂

File: rules/core/color-hex-length/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#### 📍 color-hex-length
+
+Forces the length of hex codes used in styles to be long notation rather than short.
+
+##### 🧟 Example of incorrect code for this rule:
```
Comment (rowbi-wan) [inline rules/core/color-hex-length/README.md]: Please change to new file structure. See https://github.com/netsells/stylelint-config-netsells/pull/17 for an example.

**Review by rowbi-wan — CHANGES_REQUESTED** [review]

**Comment by samturrell** [comment]
@BeccaAnderton18 Can you convert to new format and change to warning pls :D I don't think anything should be error except things like `!important` or syntax error catchers

**Review by samturrell — APPROVED** [review]

**Pull request merged**